### PR TITLE
Removes all manual parts, mentions Thyra

### PR DESCRIPTION
**Pull request by SlnPons**

### CODE REVIEW

**Review by longgamma-massa — CHANGES_REQUESTED** [review]
1. Add proper links
2. Check the links for Thyra installation - when I click on it I go to github and it's not clear what to do next.
3. Main check to do: after the one-click installation, how to access thyra interface? Does it open by itself in-browser? If yes, we don't need to add anything. If no, add a step to say how to open Thyra in-browser.

File: web3-dev/decentralized-web.rst
Comment (SlnPons) [reply web3-dev/decentralized-web.rst:79]: Good point, added the cmd to use to run Thyra

File: web3-dev/decentralized-web.rst
Comment (SlnPons) [reply web3-dev/decentralized-web.rst:54]: Nothing opens automatically. The way it works now is: 
you run the script using the cmd 
at the end of the install, the script is telling: "successfully installed"
Then you go back to the documentation to follow the next steps, which is: create your wallet.

That was my thinking process. So I don't ask user to open anything yet, because they'll do it right after. 
Is that clear enough?

File: web3-dev/decentralized-web.rst
Comment (SlnPons) [reply web3-dev/decentralized-web.rst:26]: You probably right. @gregLibert do you agree with Ena's comment? 
I think I like this story too much so I'm putting it everywhere, but if you both agree, let's just delete this section.

Comment (gregLibert) [reply web3-dev/decentralized-web.rst:26]: I think that the name we give to things is extremely important since it allows people to "project" themselves onto the named object.

I am of the opinion to remove this paragraph and add a "Why this name" section.
For this section I imagine something like:

As explained Thyra plays the role of a gateway to the Massa blockain. Massa being a coin minted in the city of [massalia](https://en.wikipedia.org/wiki/Massalia), the ancient marseille founded by the greek, quite naturaly the name [θύρα](https://en.wiktionary.org/wiki/%CE%B8%CF%8D%CF%81%CE%B1) (entrance, front door in ancient greek) imposed itself.
In order to simplify and standardize its writing, we transformed it into Thyra, but its pronunciation [\\tʰý.raː\\](https://www.youtube.com/watch?v=_0BQ7sSJMTw) remained.

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: web3-dev/decentralized-web.rst
Comment (SlnPons) [reply web3-dev/decentralized-web.rst]: Thank you guys. I understand your points, i've updated the doc accordingly.

**Review by longgamma-massa — CHANGES_REQUESTED** [review]

**Review by gregLibert — COMMENTED** [review]
:clap: good job!

**Review by gregLibert — APPROVED** [review]

**Comment by gregLibert** [comment]
@longgamma-massa waiting for your approval before behind able to merge.

**Review by longgamma-massa — APPROVED** [review]

**Pull request merged**
